src/esp/sensor/Sensor.cpp

```diff
@@ -47,12 +49,12 @@ void Sensor::setTransformationFromSpec() {
     return;
   }
 
-  resetTransformation();
+  node().resetTransformation();
 
-  translate(spec_->position);
-  rotateX(spec_->orientation[0]);
-  rotateY(spec_->orientation[1]);
-  rotateZ(spec_->orientation[2]);
+  node().translate(Magnum::Vector3(spec_->position));
+  node().rotateX(Magnum::Rad(spec_->orientation[0]));
```

Comment:
I believe these should be in degrees

Reply:
I actually had to look twice to be sure -- the (now deleted) `AttachedObject::rotateX()` had a parameter `angleInRad` and there was another `rotateXInDegree()` (which is not used here).